app/src/main/kotlin/com/wire/android/ui/home/conversationslist/ConversationRoute.kt

```diff
@@ -6,22 +6,21 @@ import androidx.compose.runtime.Composable
 import androidx.compose.runtime.collectAsState
 import androidx.compose.runtime.getValue
 import androidx.compose.ui.res.stringResource
+import androidx.hilt.navigation.compose.hiltViewModel
 import androidx.navigation.compose.NavHost
 import androidx.navigation.compose.composable
 import androidx.navigation.compose.rememberNavController
 import com.wire.android.R
 import com.wire.android.ui.common.FloatingActionButton
 import com.wire.android.ui.common.WireBottomNavigationBar
 import com.wire.android.ui.common.WireBottomNavigationItemData
-import com.wire.android.ui.main.conversationlist.ConversationState
-import com.wire.android.ui.main.conversationlist.MentionScreen
 import com.wire.android.ui.main.conversationlist.navigation.ConversationsNavigationItem
 
 
 @OptIn(ExperimentalMaterial3Api::class)
 @Composable
-fun ConversationRoute(viewModel: ConversationViewModel = ConversationViewModel()) {
-    val uiState by viewModel.state.collectAsState()
+fun ConversationRoute(conversationListViewModel: ConversationListViewModel = hiltViewModel()) {
```

Comment:
Can we rename to `ConversationRouter`? This composable is not really a route but rather a wrapper that helps routing to the specific conversation screen